docs/modding/suitmodding/addinglocals.mdx

```diff
@@ -0,0 +1,666 @@
+<div id="langcode"></div>
+```json
+"localisationOverrides": {
+        "yourLocrHASH": {
+            "english": {
+                "yourconverteddecimal": "New Suit Name"
+            },
+            "french": {
+                "yourconverteddecimal": "Niveua Costume nome"
+            }
+        }
+    }
+```
+<details>
+<summary>Language Codes</summary>
+
+:::info language codes
+- The reason why `french` is used here and not `français` is because the manifest's <Tooltip text={<strong>schema</strong>} tooltip="A structure used to validate and provide auto-completion for your manifest, helping you write error-free code." /> is coded in English. 
+- As for why it doesn't use the same <Tooltip text={<strong>language codes</strong>} tooltip="The language identifiers in each language array of a LOCR file" /> as the actual <Tooltip text={<strong>LOCR Files</strong>} tooltip="A file that holds game text in different languages, linked by unique 'strings'." /> like `en` or `fr`, is because the manifest's <Tooltip text={<strong>schema</strong>} tooltip="A structure used to validate and provide auto-completion for your manifest, helping you write error-free code." /> is coded like that. 
+
+To see how all languages have to be written down in the <Tooltip text={<strong>manifest</strong>} tooltip="A file in your mod folder that holds important information and can execute code" />, see the <Tooltip text={<strong>manifest documentation in SMF</strong>} tooltip="It's the book icon on the left after enabling developer mode." />. 
+Or *[click here](#all-languages)*
+:::
+</details>
+### 6. Add the description
+*Now try the same for the description as well.*
+
+- <Tooltip text={<strong>convert</strong>} tooltip="Using the Localisation Hash Calculator box" /> the <Tooltip text={<strong>string</strong>} tooltip= "A key that links to text in the LOCR File" /> from the <Tooltip text={<strong>suit's description</strong>} tooltip="Usually right below the string of your suit's name" />
+- add your new <Tooltip text={<strong>converted decimal</strong>} tooltip="A string converted into a numeral form" /> in the manifest with the new text 
+- *(Optionally)* add another language for the new description as well
+
+Example:
+```json
+"localisationOverrides": {
+        "yourLocrHASH": {
+            "english": {
+                "yourconverteddecimal": "New Suit Name",
+                "yourotherconverteddecimal": "New Suit Description"
+            },
+            "french": {
+                "yourconverteddecimal": "Niveua Costume nome",
+                "yourotherconverteddecimal": "Description du nouveau costume"
+            }
+        }
+    }
+```
+>*This recreates how the* <Tooltip text={<strong>LOCR File</strong>} tooltip="A file that holds game text in different languages, linked by unique 'strings'." /> *is internally set up by using the same* <Tooltip text={<strong>syntax</strong>} tooltip="A set of rules that define how code is written and organized." />.
```

Comment:
While it would be nice if it did, the manifest does not use the same syntax as the LOCR file, that's why you've had to explain how to use decimal hashes and that it's `english` not `en`